workspaces/report-portal/plugins/report-portal/src/components/ReportPortalOverviewCard/ReportPortalOverviewCard.tsx

```diff
@@ -0,0 +1,241 @@
+      aria-hidden="true"
+    >
+      {props.children}
+    </StyledTypography>
+  );
+};
+
+const StyledResults = styled(Typography)({
+  '& > *': { fontWeight: '800' },
+});
+
+export const ReportPortalOverviewCard = (props: {
+  variant: InfoCardVariants;
+}) => {
+  const config = useApi(configApiRef);
+  const hostsConfig = config.getConfigArray('reportPortal.integrations');
+
+  const { entity } = useEntity();
+  const projectId =
+    entity.metadata.annotations?.['reportportal.io/project-name'] ?? '';
+  const launchName =
+    entity.metadata.annotations?.['reportportal.io/launch-name'] ?? '';
+  const hostName =
+    entity.metadata.annotations?.['reportportal.io/host'] ??
+    hostsConfig[0].getString('host');
+
+  const [defects, setDefects] = useState<Defect[]>([]);
+  const [filters, _] = useState<{ [key: string]: string | number } | undefined>(
+    {
+      'filter.eq.name': launchName,
+    },
+  );
+
+  const { loading, launchDetails } = useLaunchDetails(
+    projectId,
+    hostName,
+    filters,
+  );
+  const { loading: projectLoading, projectDetails } = useProjectDetails(
+    projectId,
+    hostName,
+  );
+
+  useEffect(() => {
+    if (!loading && launchDetails && projectDetails) {
+      const tempArr: Defect[] = [];
+      Object.keys(launchDetails.statistics.defects).forEach(defect => {
+        tempArr.push({
+          name: projectDetails.configuration.subTypes?.[defect.toUpperCase()][0]
+            .longName,
+          total: launchDetails.statistics.defects?.[defect].total,
+          color:
+            projectDetails.configuration.subTypes?.[defect.toUpperCase()][0]
+              .color,
+          id: projectDetails.configuration.subTypes?.[defect.toUpperCase()][0]
+            .id,
+        });
+      });
+      setDefects(tempArr);
+    }
+  }, [loading, launchDetails, projectDetails]);
+
+  if (!isReportPortalAvailable(entity)) return null;
+
+  return (
+    <InfoCard
+      title={
+        <HeaderComponent
+          total={launchDetails?.statistics.executions.total ?? -1}
+        />
+      }
+      variant={props.variant}
+      divider
+      deepLink={{
+        link: `https://${hostName}/ui/#${projectId}/launches/latest/${launchDetails?.id}`,
+        title: 'View on Report Portal',
+      }}
+    >
+      <Grid container spacing={3}>
+        {loading ? (
+          <>
+            <Grid item xs={12}>
+              <Skeleton animation="wave" />
+            </Grid>
+            <Grid item xs={4}>
+              <Skeleton animation="wave" />
+            </Grid>
+            <Grid item xs={4}>
+              <Skeleton animation="wave" />
+            </Grid>
+            <Grid item xs={4}>
+              <Skeleton animation="wave" />
+            </Grid>
+          </>
+        ) : (
+          <>
+            <Grid item xs={12}>
+              <MultiProgress
+                transitionTime={3}
+                elements={[
+                  {
+                    color: '#28b463',
```

Comment:
There are several hard coded colors here, is there no option to pull these from the theme?

Reply:
these colors are Red, Green, Gray for passed, failed and skipped test cases, I don't think its available in backstage theme palette

Reply:
These would not work?

https://github.com/backstage/backstage/blob/22b42e5b3ce3e595e6fd1945b0880de33372b860/packages/theme/src/base/types.ts#L23-L30

Reply:
updated the code, these colors are used from palette now